src/jackknife.jl

```diff
@@ -0,0 +1,163 @@
+"""
+    Delete 1 jackknife with  for unstratified designs.
+    
+    Replicate weights are given by:
+    wi(hj)  = 0, if observation unit i is in psu j of stratum h
+            = nh(nh −1)wi, if observation unit i is in stratum h but not in psu j.
+    ## Reference
+    pg 380-382, Section 9.3.2 Jackknife - Sharon Lohr, Sampling Design and Analysis (2010)
+"""
+function jk1weights(design::SurveyDesign)
+    cluster_sorted = sort(design.data, design.cluster) # Does jk need sorting?
+    psus = unique(cluster_sorted[!, design.cluster])
+    nh = length(psus)
```

Comment:
Just for consistency of notation: `nh` in the paper refers to the number of PSUs sampled in cluster `h`. Should we use a different notation for this?